fvm/transaction.go

```diff
@@ -71,11 +72,13 @@ func (proc *TransactionProcedure) Run(vm *VirtualMachine, ctx Context, st *state
 
 type TransactionInvocator struct {
 	logger zerolog.Logger
+	checkStorage bool
 }
 
-func NewTransactionInvocator(logger zerolog.Logger) *TransactionInvocator {
+func NewTransactionInvocator(logger zerolog.Logger, checkStorage bool) *TransactionInvocator {
```

Comment:
I think we should keep the params like checkStorage as part of the context which are passed to the Process.